torch_frame/datasets/huggingface_dataset.py

```diff
@@ -0,0 +1,90 @@
+from __future__ import annotations
+
+import pandas as pd
+
+import torch_frame
+from torch_frame import stype
+from torch_frame.utils.split import SPLIT_TO_NUM
+
+
+class HuggingFaceDatasetDict(torch_frame.data.Dataset):
+    r"""Load a Hugging Face :obj:`datasets.DatasetDict` dataset
+    to a :obj:`torch_frame.data.Dataset` with pre-defined split information.
+
+    Args:
+        path (str): Path or name of the dataset.
```

Comment:
name

Reply:
"path" is the one in the HF dataset https://huggingface.co/docs/datasets/v2.15.0/en/package_reference/loading_methods#datasets.load_dataset